examples/multipart_form.rs

```diff
@@ -21,7 +21,7 @@ async fn main() {
     // run it with hyper
```

Comment:
Maybe also standardize these comments across the examples? :)

Reply:
Sure!